adapters/Maio/MaioAdapter/RTBAdapter/GADRTBMaioInterstitialAd.m

```diff
@@ -0,0 +1,101 @@
+//
+//  GADRTBMaioInterstitialAd.m
+//  Adapter
+//
+//  Created by i-mobile on 2020/11/18.
+//  Copyright © 2020 Google. All rights reserved.
+//
+
+#import "GADRTBMaioInterstitialAd.h"
+#import "GADMMaioConstants.h"
+#import <stdatomic.h>
+
+#import <MaioOB/MaioOB-Swift.h>
+
+#define MaioInterstitial MaioRewarded
+#define MaioInterstitialLoadCallback MaioRewardedLoadCallback
+#define MaioInterstitialShowCallback MaioRewardedShowCallback
+
```

Comment:
Are these needed? Why do we need this wrapper? 
If they use the rewarded objects and Maio doesn't actually have an interstitial API, let's just call them what they are.